### PR TITLE
Italian translation for 1.3.0

### PR DESCRIPTION
**Pull request by jodosha**
:it: :rainbow: :heart: :green_heart: :yellow_heart: :blue_heart:

/cc @amencarini and @alepore for review

### CODE REVIEW

**Comment by alepore** [comment]
Looks good to me! :+1: 

Some minor things:
- some commas can probably be removed? ex. the last of line 3, line 18
- non-professionali => non professionali ?
- tickets => ticket ?

**Comment by jodosha** [comment]
@alepore Thank you. Done.

**Comment by CoralineAda** [comment]
@jodosha Can you also update the .txt and .htm files? Then we should be ready to merge.

File: version/1/3/0/it/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Con l'adozione di questo Codice di Condotta i responsabili del progetto si impegnano ad applicare equamente e costantemente questi princìpi ad ogni aspetto della gestione di questo progetto. I responsabili del progetto che non seguiranno o applicheranno il Codice di Condotta potranno essere permanentemente rimossi dal team.
+
+Questo Codice di Condotta è applicabile sia al progetto online che agli spazi pubblici quando un individuo rappresenta il progetto stesso o la sua comunità.
+
+Casi di comportamento ingiurioso, molestante o altrimenti inaccettabile possono essere riportati contattando il responsabile del progetto tramite [INSERIRE INDIRIZZO EMAIL]. Tutti i reclami saranno revisionati ed indagati e risulteranno in una risposta ritenuta necessaria ed appropriata alle circostanze. I responsabili sono obbligati a manterere riserbo rispetto a chi riporta un caso.
+
+Questo Codice di Condotta è adattado da [Contributor Covenant](http://contributor-covenant.org), versione 1.3.0, disponibile presso [http://contributor-covenant.org/version/1/3/0/](http://contributor-covenant.org/version/1/3/0/)
```
Comment (CoralineAda) [inline version/1/3/0/it/code_of_conduct.md]: This should point to /version/1/3/0/it/

**Comment by brassy-** [comment]
**Minus**
* Instead of ‘trolling’, that in this case is a word loan, I suggest ‘comportamento da troll’.
* Instead of ‘molestante’, I suggest to use the adjective ‘molesto’ 

**General**
Why translate ‘code of conduct’ with ‘codice di condotta’? I suggest ‘codice di comportamento’, if you need to translate it.

File: version/1/3/0/it/code_of_conduct.md (hunk 1 above)
Comment (proudlygeek) [inline version/1/3/0/it/code_of_conduct.md]: Hi, there's a small typo: adatta**d**o should be adatta**t**o

**Comment by jodosha** [comment]
@CoralineAda Done.

**Pull request merged**